test/ngResource/resourceSpec.js

```diff
@@ -1312,102 +1312,221 @@ describe('basic usage', function() {
       });
     });
 
-    it('should allow per action response interceptor that gets full response', function() {
-      CreditCard = $resource('/CreditCard', {}, {
-        query: {
-          method: 'get',
-          isArray: true,
-          interceptor: {
-            response: function(response) {
-              return response;
-            }
+
+    describe('responseInterceptor', function() {
+      it('should allow per action response interceptor that gets full response', function() {
```

Comment:
"full response" here means no headers or statusText, right? Because you don't assert for them. Could we add them if they are missing?